srcpkgs/outguess/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'outguess'
+pkgname=outguess
+version=0.4
+revision=1
+build_style=gnu-configure
+configure_args="--with-generic-jconfig"
+hostmakedepends="automake"
+short_desc="Universal steganographic tool"
+maintainer="Eloi Torrents <eloitor@disroot.org>"
+license="custom:Custom"
```

Comment:
This is `BSD-4-Clause`, debian uses that too https://metadata.ftp-master.debian.org/changelogs//main/o/outguess/outguess_0.2.2-5_copyright.